cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,145 @@
+
+    with torch.no_grad():
+        for index, batch in enumerate(dataloader):
+            input_data = batch
+            input_ids = input_data['input_ids'].to(DEVICE)
+            attention_mask = input_data['attention_mask'].to(DEVICE)
+            
+            outputs = model(input_ids=input_ids, attention_mask=attention_mask)
+
+    output_decoded = decode_output(outputs)
+    cvss_string = generate_cvss_string(output_decoded)
+    return cvss_string
```

Ruff

Unnecessary assignment to `cvss_string` before `return` statement

Remove unnecessary assignment

(RET504)